src/utils/file.test.ts

```diff
@@ -0,0 +1,27 @@
+import { similarityPerc } from './file';
+
+describe('with two files', () => {
+  it('calculate similarityPerc of chars changed big text', async () => {
+    const dperc = similarityPerc(
+      'src/rules/test-cases/general/modules/mod1-js/filediff1a',
+      'src/rules/test-cases/general/modules/mod1-js/filediff1b',
+    );
+    expect(dperc).toEqual(90.33);
+  });
+
+  it('calculate similarityPerc of chars changed big text rev', async () => {
+    const dperc = similarityPerc(
+      'src/rules/test-cases/general/modules/mod1-js/filediff1b',
+      'src/rules/test-cases/general/modules/mod1-js/filediff1a',
+    );
+    expect(dperc).toEqual(89.3);
```

Comment:
Maybe instead of putting "equal" could test against the reference threshold? Like, if the threshold is 80, you check if it is `>80`, this way, if we update the similarity lib, or even change it, we don't have to update this test just because the result after thise update/change is now `90.12` instead of `90.33`.
wdty?